eng/common/Update-Change-Log.ps1

```diff
@@ -139,6 +139,7 @@ if (Test-Path -Path $ChangeLogPath -PathType Container)
 
 # Read current change logs and add/update version
 $ChangelogLines = [System.Collections.ArrayList](Get-Content -Path $ChangeLogPath)
+if ($null -eq $ChangelogLines) { $ChangelogLines = @() }
```

Comment:
nit: I realize there's one line for the if statement but can we format this if like the rest of the if statements in the file? Since this is only a format change I'm going to pre-approve.